next-tavla/src/Board/scenarios/Table/index.tsx

```diff
@@ -15,48 +15,44 @@ const columnComponents: Record<TColumn, () => JSX.Element> = {
     line: Line,
     time: Time,
     platform: Platform,
-    via: Via,
     situations: Situations,
+    via: Via,
 }
 
-function flexToPercentage(columnSettings: TColumnSetting[]) {
-    const flexToPercentage = columnSettings.reduce(
-        (acc, val) => acc + (val.size ?? 1),
-        0,
-    )
-
-    const perc = 100 / flexToPercentage
+const ColumnOrder: TColumnOrder = [
+    { type: 'line', size: 1 },
+    { type: 'destination', size: 2 },
+    { type: 'via', size: 2 },
+    { type: 'platform', size: 3 },
+    { type: 'situations', size: 4 },
+    { type: 'time', size: 1 },
+]
 
-    return columnSettings.map((set) => ({
-        ...set,
-        size: (set.size ?? 1) * perc,
-    }))
+const columns = DefaultColumns
+function TH({ type, size }: { type: TColumn; size: number }) {
+    return (
+        <th
+            style={{
+                width: size,
+            }}
+        >
+            {Columns[type]}
+        </th>
+    )
 }
 
-function Table({
-    columns = [...DefaultColumns],
-    departures,
-}: {
-    columns?: TColumnSetting[]
-    departures: TDepartureFragment[]
-}) {
-    const columnSizes = flexToPercentage(columns)
-
+function Table({ departures }: { departures: TDepartureFragment[] }) {
     return (
         <div className={classes.container}>
             <table className={classes.table}>
                 <thead>
                     <tr>
-                        {columnSizes.map((col) => (
-                            <th
-                                style={{
-                                    width: col.size,
-                                }}
-                                key={col.type}
-                            >
-                                {Columns[col.type]}
-                            </th>
-                        ))}
+                        {ColumnOrder.map(
+                            ({ type, size }) =>
+                                columns[type] && (
```

Comment:
Kan vi filtrere ut kolonnene vi ikke skal ha med fra listen på forhånd istedet for denne inline sjekken.